sdk/python/lib/test/test_invoke.py

```diff
@@ -89,3 +90,17 @@ def test_invoke_empty_return(tok: str, version: str, empty: bool, expected) -> N
     props = {"empty": True} if empty else {}
     opts = pulumi.InvokeOptions(version=version) if version else None
     assert pulumi.runtime.invoke(tok, props, opts).value == expected
+
+
+@pytest.mark.parametrize(
+    "a,b,expected",
+    [
+        (InvokeOptions(version="1.0.0"), InvokeOptions(version="2.0.0"), "2.0.0"),
+        (None, InvokeOptions(version="2.0.0"), "2.0.0"),
+        (InvokeOptions(version="1.0.0"), None, "1.0.0"),
```

Comment:
```suggestion
        (InvokeOptions(version="1.0.0"), None, "1.0.0"),
        (InvokeOptions(version="1.0.0"), InvokeOptions(version=""), ""),
```

Might be worth adding this case,  now that we decided that the right fix is to set the default to None (which I agree with)

Reply:
Good idea, thanks!